lib/livebook/hubs/team_client.ex

```diff
@@ -837,25 +837,25 @@ defmodule Livebook.Hubs.TeamClient do
     put_in(state.hub, hub)
   end
 
-  defp put_billing_status(hub, %LivebookProto.BillingStatus{} = status) do
+  # TODO: Remove when Billing is public
+  defp put_billing_status(hub, _status) do
     put_in(
       hub.billing_status,
-      Map.merge(%{disabled: status.disabled}, put_billing_status(status.type))
+      %{disabled: false, type: nil}
     )
   end
 
-  # TODO: Remove when Billing is public
-  defp put_billing_status(hub, nil) do
+  defp put_billing_status(hub, %LivebookProto.BillingStatus{} = status) do
     put_in(
       hub.billing_status,
-      %{disabled: false, type: nil}
+      Map.merge(%{disabled: status.disabled}, put_billing_status(status.type))
     )
   end
 
   defp put_billing_status(
          {:trialing,
           %LivebookProto.BillingStatusTrialing{
-            trial_ends_at: %DateTime{} = trial_ends_at
+            trial_ends_at: trial_ends_at
           }}
        ) do
     %{type: :trialing, trial_ends_at: DateTime.from_unix!(trial_ends_at)}
```

Comment:
> ```
> trial_ends_at: %DateTime{} = trial_ends_at
> ...
> DateTime.from_unix!(trial_ends_at)
> ```

this was clearly non-sensical, thank you for fixing this. CI caught a bug, after that is fixed, ship it!